atsign-foundation/at_wavi_app
# fixes #162 (Enhance Profile Sharing)

**- What I did**
     Earlier when we choose to share profile we were taken to the wavi.ng where we can see the sharing options. I added one screen for showing QR code to share the profile. also added a button named share via app which will prompt user to select any app for sharing the user's profile. Also in the platform's share dialog, we have a "copy to clipboard" option.

**- How I did it**
      I have used 2 packages, qr_flutter which takes data as a string and shows the QR code, share_plus - a Flutter plugin to share content from the Flutter app via the platform's share dialog. 

**- How to verify it**
      To verify the QR code, we have to scan one person's QR code from another person's wavi app. To verify the share via app button, we can share our profile using various apps shown in the share dialog.

**- Description for the changelog**
      Implemented QR code feature for sharing the user profile and added option to share via any app e.g. WhatsApp, Gmail, LinkedIn, etc.
